en-US/firefox-ios.xliff

```diff
@@ -3232,6 +3237,11 @@
         <target>%d tab(s) closed</target>
         <note>The label indicating that all the tabs were closed</note>
       </trans-unit>
+      <trans-unit id="Tabs.DeleteAllUndo.Title.v113" xml:space="preserve">
+        <source>%d Tab(s) Closed</source>
+        <target>%d Tab(s) Closed</target>
+        <note>The label indicating that all the tabs were closed</note>
```

Comment:
Comment should explain what placeholder will be replaced with.
Also, this string assumes that all languages use plural the same way as English (only two forms), which they do not.
Since we don't support proper plurals at the time, a workaround would be to format the string as "Tabs closed: %d" (just leave "tabs" as plural, even if it's not ideal)
No need to update string ID since strings haven't been exposed yet